src/content/blog/astro-320.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: "Astro 3.2: View Transitions improvements"
+description: "Astro 3.2 is out with several improvements to View Transitions, including the ability to control the history stack, and a JavaScript navigation API"
+publishDate: "September 28, 2023"
+authors:
+  - matthew
+  - martrapp
+  - chris
+coverImage: "/src/content/blog/_images/astro-320/blog-hero-3_2-release.webp"
+socialImage: "/src/content/blog/_images/astro-320/og-image-3_2-release.webp"
+lang: "en"
+---
+
+Astro 3.2 is out and brings several new improvements for [view transitions](https://docs.astro.build/en/guides/view-transitions/) and nested integrations:
```

Comment:
would phrase to be a bit more clear on the “why” vs the “what”. Something like:

“Astro 3.2 is out with several new improvements that make view transitions and integrations even easier to use:”